web-app/js/portal/ui/openlayers/SpatialConstraintMap.js

```diff
@@ -6,6 +6,7 @@ OpenLayers.SpatialConstraintMap = OpenLayers.Class(OpenLayers.Map, {
 
```

Comment:
I was confused by the names used in this file. The use of `polyonStyle` was just too weird.
Suggest renaming of variables like:
```
    setDefaultSpatialConstraintType: function(constraintType) {
        this.defaultSpatialConstraintType = constraintType;
        this.setSpatialConstraint();
    },

    setSpatialConstraintType: function(constraintType) {

        // Avoid unnecessary removal/addition of the control.
        if (this.constraintType != constraintType) {
            this.doSetSpatialConstraint(constraintType);
            this.events.triggerEvent('spatialconstrainttypechanged', constraintType);

            var val = 'type=' + constraintType;
            trackFiltersUsage('filtersTrackingSpatialConstraintAction', val);
        }
    },
```